ui/jobs/components/index.ts

```diff
@@ -376,4 +357,33 @@ export class ComponentManager {
     )
       this.bars.setJobsContainerVisibility(false);
   }
+
+  private _shouldShowFoodBuff(): void {
+    if (!this._canShowFoodBuff()) {
+      this.bars._showFoodBuff(false);
+      return;
+    }
+
+    const showAtMs = this.foodBuffExpiresTimeMs - (this.options.HideWellFedAboveSeconds * 1000);
+    const showMs = showAtMs - Date.now();
+
+    window.clearTimeout(this.foodBuffTimer);
+    this.foodBuffTimer = 0;
+
+    if (showMs <= 0) {
+      this.bars._showFoodBuff(true);
+    } else {
+      this.bars._showFoodBuff(false);
+      this.foodBuffTimer = window.setTimeout(this._shouldShowFoodBuff.bind(this), showMs);
```

Comment:
I know this was here before, but this should probably use an arrow function instead of a `bind`:
```suggestion
      this.foodBuffTimer = window.setTimeout(() => this._shouldShowFoodBuff(), showMs);
```